ibc-core/ics02-client/types/src/status.rs

```diff
@@ -53,6 +53,14 @@ impl Status {
             status => Err(ClientError::ClientNotActive { status }),
         }
     }
+
+    /// Checks whether the client is either frozen or expired; returns `Err` if not.
+    pub fn verify_is_inactive(self) -> Result<(), ClientError> {
```

Comment:
also, it should be `&self`. Same for `verify_is_active`.

```rs
    pub fn verify_is_inactive(&self) -> Result<(), ClientError> {
        match self {
            Self::Frozen | Self::Expired => Ok(()),
            &status => Err(ClientError::ClientNotInactive { status }),
        }
    }
```

Reply:
[eee31c6](https://github.com/cosmos/ibc-rs/pull/1127/commits/eee31c63c1f2e315639acfb15c3e47a0d1c43172)

Reply:
I don't think we need `clone` here. See `&status` match instead of `status`. `Status` should have `Copy` implemented.

Reply:
[f7a9a6b](https://github.com/cosmos/ibc-rs/pull/1127/commits/f7a9a6b7d2ae705f828fe78a9126524218420ee3)